packages/forma-36-website/src/components/Container.js

```diff
@@ -49,34 +49,34 @@ const styles = {
 };
 
 const markToComponentMap = {
-  h1: props => <DisplayText {...props} />,
-  h2: props => <Heading element="h2" {...props} />,
-  h3: props => <Subheading element="h3" {...props} />,
-  h4: props => <Subheading element="h4" {...props} />,
-  h5: props => <Subheading element="h5" {...props} />,
-  h6: props => <Subheading element="h6" {...props} />,
-  p: props => <Paragraph className="f36-font-size--l" {...props} />,
-  a: props => <TextLink {...props} />,
-  ul: props => <List className="f36-margin-bottom--m" {...props} />,
-  li: props => (
+  h1: (props) => <DisplayText {...props} />,
+  h2: (props) => <Heading element="h2" {...props} />,
+  h3: (props) => <Subheading element="h3" {...props} />,
+  h4: (props) => <Subheading element="h4" {...props} />,
+  h5: (props) => <Subheading element="h5" {...props} />,
+  h6: (props) => <Subheading element="h6" {...props} />,
+  p: (props) => <Paragraph className="f36-font-size--l" {...props} />,
+  a: (props) => <TextLink style={{ fontSize: tokens.fontSizeL }} {...props} />,
```

Comment:
Link the same size as Paragraph, but I think we need to do something global maybe if we really want to overwrite font-size. I am not sure... Maybe we would just keep 14px, so the default for Paragraph?